test/apps/test_init_mt.c

```diff
@@ -50,6 +61,10 @@ int main(int argc, char **argv)
 
 #pragma omp barrier
 
-    printf("finished %d threads\n", count);
-    return 0;
+    gettimeofday(&finish, NULL);
+    printf("[%ld.%06ld] ", finish.tv_sec, finish.tv_usec);
```

Comment:
add \n

Reply:
there was idea 1 run - 1 line to simplify parsing of logs